### ISSUE TITLE
Scale grid without changing content zoom

### ISSUE THREAD

**Issue by xuanswe** [tldraw/tldraw#3620]
### What's the feature?

Currently, a background grid could be enabled under "Preferences > Show grid."
The default grid size is quite nice, I like it.
But sometime, it's still a bit smaller or bigger than the size I need.
This is quite frustrating under a **small pen graphic tablet**.

It would be nice if it is possible to scale (in/out) only the grid without changing the current content zoom level.
I think this feature could be a menu item right below "Preferences > Show grid."

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by dimateos** [comment]
Yeah grid customization would be great. I also agree that grid zoom would be the most basic customization to support first.

There could also be additional grid patterns like for isometric drawing etc. Here there is a discussion on excalidraw which could be applicable for tldrwaw too https://github.com/excalidraw/excalidraw/issues/6773

Note that correct snapping implemented for all would be complex but sometimes a visual reference suffices, hence even a custom user image to set as background with simple scaling features would be a great feature.

**Comment by dimateos** [comment]
Just in case I am unable to reopen once closed: /keepalive
